docs/quick_start/deploy_in_docker.md

```diff
@@ -1,44 +1,121 @@
-## Introduction
+# Quick start: Deploy StarRocks with Docker
 
-This document is used to deploy the simplest StarRocks cluster, which contains one FE and one BE.
+This quickstart provides a guide to:
+* Use [Docker](https://docs.docker.com/engine/install/) to deploy StarRocks with one FE and one BE.
+* Connect to StarRocks with MySQL client
+* Create a table, insert some data and query it.
 
 ## Prerequisites
 
-1. docker
-2. mysql client
+1. Docker
+2. MySQL client
 
-## Step 1: Get docker-compose file
-
-You need to download the docker compose yaml file of starrocks.
+## Step 1: Deploy
 
 ```sh
-wget https://raw.githubusercontent.com/StarRocks/starrocks/main/docker/docker-compose/docker-compose.yml
+docker run -p 9030:9030 -p 8030:8030 -p 8040:8040 -itd starrocks/allin1-ubuntu:2.5.4
 ```
 
-## Step 2: Run StarRocks cluster
+Then you can check the container status with:
+```sh
+docker ps
+```
 
-You should run the following command
+## Step 2: Connect to StarRocks
 
+StarRocks needs some time to get ready, it is recommended to wait at least 30 seconds before connecting.
 ```sh
-docker-compose up -d
+mysql -P9030 -h127.0.0.1 -uroot --prompt="StarRocks > "
 ```
 
-Then you can check the containers status.
+## Step 3: Using StarRocks
 
-## Step 3: Connect to StarRocks
+Use the following commands to check the status of FE and BE. If  `Alive` shows true for both FE and BE, StarRocks is healthy and ready to go.
 
-Connect to StarRocks cluster
+FE
+```SQL
+SHOW PROC '/frontends'\G
+```
+```plaintext
+StarRocks > SHOW PROC '/frontends'\G
+*************************** 1. row ***************************
+             Name: be552e5f0de9_9010_1680659932444
+               IP: be552e5f0de9
+      EditLogPort: 9010
+         HttpPort: 8030
+        QueryPort: 9030
+          RpcPort: 9020
+             Role: LEADER
+        ClusterId: 1556630880
+             Join: true
+            Alive: true
+ReplayedJournalId: 944
+    LastHeartbeat: 2023-04-05 02:49:36
+         IsHelper: true
+           ErrMsg: 
+        StartTime: 2023-04-05 01:59:00
+          Version: 3.0.0-RC01-a99fb8571
```

Comment:
this line tells the truth that your example is running with `3.0.0-rc01`. better switch to `2.5.4`. :)